README.md

```diff
@@ -182,3 +182,21 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
 - [minitox](https://github.com/hqwrong/minitox) (A minimal tox client)
+
+## SAST Tools
```

Comment:
Never heard of "SAST", is it a common acronym that I somehow missed or, if it's not very common, then perhaps we should also define what it is?